fe/fe-core/src/main/java/com/starrocks/transaction/TransactionState.java

```diff
@@ -121,6 +121,9 @@ public static TxnStatusChangeReason fromString(String reasonString) {
                 if (reasonString.contains(txnStatusChangeReason.toString())) {
                     return txnStatusChangeReason;
                 }
+                if (reasonString.contains("Unrecoverable error: ")) {
```

Comment:
https://github.com/rickif/starrocks/blob/f6738f3b1555931df1a51245a487599716238c3f/fe/fe-core/src/main/java/com/starrocks/transaction/TransactionState.java#L114
The `TStatusCode` may be not OK here, since the code info is removed in BE. So I add a enum here.